plugins/woocommerce/src/Admin/Features/PaymentGatewaySuggestions/DefaultPaymentGateways.php

```diff
@@ -966,8 +966,8 @@ public static function get_rules_selling_offline() {
 				),
 			),
 			'option_name'  => 'woocommerce_onboarding_profile',
-			'operation'    => 'contains',
-			'value'        => 'no_im_selling_offline',
+			'operation'    => 'in',
+			'value'        => array( 'no_im_selling_offline', 'aim_selling_both_online_and_offline' ),
```

Comment:
```suggestion
			'value'        => array( 'no_im_selling_offline', 'im_selling_both_online_and_offline' ),
```

Reply:
Thanks for catching that! I missed pushing my last commit 😅